app/helpers/application_helper.rb

```diff
@@ -93,4 +93,30 @@ def service_for(previous_url, current_user)
       url: url,
     }
   end
+
+  def footer_navigation
+    navigation = []
+    t("footer_sections").each do |section|
+      nav_section = section
+      nav_section[:items] = []
+      section[:links].each do |link|
+        item = link
+        # use link text for data-track-label unless a different label is specified
+        item[:attributes][:data][:track_label] = link[:attributes][:data][:track_label] || link[:text]
+        nav_section[:items] << item
+      end
+      navigation << nav_section
```

Comment:
Nitpick: I'd put an empty newline before line 108 for consistency and readability (you're already doing it on line 119).